HaikuPorter/SourceFetcher.py

```diff
@@ -273,20 +290,29 @@ def __init__(self, uri, fetchTarget):
 		(unusedType, self.uri, self.rev) = parseCheckoutUri(uri)
 		if not self.rev:
 			self.rev = 'HEAD'
+		if self.rev.startswith('tag=') or self.rev.startswith('commit='):
+			self.rev=self.rev[self.rev.find('=') + 1:]
+			self.sourceShouldBeValidated = True
 
 	def fetch(self):
-		if not Configuration.shallAllowUnsafeSources():
-			sysExit('Downloading from unsafe sources is disabled in ' +
-					'haikuports.conf!')
+		if not self.sourceShouldBeValidated:
```

Comment:
Hmm, what happens if we have this set, but then there's no checksum? Won't it then allow the source through without validating the checksum...?

Reply:
When there is no checksum, this is checked at https://github.com/haikuports/haikuporter/blob/5e5a2b4b49382c84b5154e508b911fff6bda74d1/HaikuPorter/Source.py#L274

Reply:
OK, makes sense then.